docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "WebLogic Domain"
+date: 2019-10-01T14:32:31-05:00
+weight: 2
+description: "WebLogic domains and operator's logs"
+---
+
+
+#### Overview
+This page describes to how to configure a WebLogic domain to use fluentd to send log information to Elasticsearch.
+
+The general mechanism of how this works is:
+
+* `fluentd` runs as a separate container in the admin and managed server pods
+* The log files reside on a volume that is shared between the `weblogic-server` and `fluentd` containers
+* `fluentd` tails the domain logs files and exports to Elasticsearch
+* A `ConfigMap` contains the filter and format rules for exporting log records
+
+For sample purposes, this document will assume a domain with the following attributes are being configured:
+
+* Domain name is `bobs-bookstore`
+* Kubernetes namespace is `bob`
+* Kubernetes secret is `bobs-bookstore-weblogic-credentials`
+* Elasticsearch configuration
+```text
+    elasticsearchhost: elasticsearch.bobs-books.sample.com
+    elasticsearchport: 443
+    elasticsearchuser: bob
+    elasticsearchpassword: changeme
+```
+
+#### Configure log files to use a volume
+The domain log files must be written to a volume that can be shared between the `weblogic-server` and `fluentd` containers.
+
+NOTE: For brevity only the paths to the relevant configuration being added is shown.  A complete example of a domain definition is at the end of this document.
+
+Example: `kubectl edit domain bobs-bookstore -n bob` and make the following edits:
+
+```yaml
+spec:
+  logHome: /scratch/logs/bobs-bookstore
+  logHomeEnabled: true
+  serverPod:
+    volumeMounts:
+    - mountPath: /scratch
+      name: weblogic-domain-storage-volume
+    volumes:
+    - emptyDir: {}
+      name: weblogic-domain-storage-volume
+```
+
+#### Add Elasticsearch secrets to WebLogic domain credentials
+The `fluentd` container will be configured to look for Elasticsearch parameters in the domain credentials.  Edit the domain credentials and add the parameters shown in the example below.
+
+Example: `kubectl edit secret bobs-bookstore-weblogic-credentials -n bob` and add the base64 encoded values of each Elasticsearch parameter:
+```text
+elasticsearchhost: ZWxhc3RpY3NlYXJjaC5ib2JzLWJvb2tzLnNhbXBsZS5jb20=
+elasticsearchport: NDQz
+elasticsearchuser: Ym9i
+elasticsearchpassword: d2VsY29tZTE=
+```
+
+## Create FluentD configuration
```

Comment:
Change heading level of Create FluentD configuration by adding two more hash marks.
FluentD -> Fluentd